README.md

```diff
@@ -122,6 +123,32 @@ Registrations are not currently supported and will be addressed in J#PROJ-988.
 - cmd: node_modules/.bin/appveyor-commit-message-checker
 ```
 
+### As a commit hook
+
+A `commit-msg` hook is provided which you can setup to run in your development environment, to help catch invalid commit
+messages as they happen.
+
+This is intended to be setup in addition to, and definitely not instead of, checks that run [as part of CI](#as-part-of-ci).
+It's primary purpose is to shorten the feedback time between commiting something, and finding out that the commit
+message is invalid. The alternative being that you don't find out until it fails on CI, which could take a frustrating
+amount of time.
+
+To use the commit hook, copy or symlink the file to `.git/hooks/commit-msg` within your project. i.e. run:
```

Comment:
Going to leave this comment as is. It does look like it's possible to have multiple scripts per hook by placing them in `commit-msg.d` folder, but I don't want to stray too much into documenting how Git works (e.g. I don't know if that's true of all version of Git).